package.json

```diff
@@ -4,47 +4,27 @@
   "description": "Client-Side SDK for Unified Data Collection",
   "main": "src/core/index.js",
   "scripts": {
+    "install": "cd sandbox && npm install",
```

Comment:
This didn't exist before, which meant a newcomer developer would need to run `npm run sandbox:install` to get sandbox functionality. Now the sandbox dependencies get installed automatically when installing the top-level dependencies.

Reply:
Doesn't this prevent you from installing the dependencies from this package.json? I think this should be put in "postinstall"

Reply:
No, it installs the dependencies from package.json and then runs the `install` script. I could have put this on `preinstall`, `install`, or `postinstall`. Post-install seems to usually be used for doing things (often using dependencies) after dependencies are installed. In this case though, we're actually still in the process of installing dependencies (the sandbox's dependencies), so that's why I figured it should be in the install stage. I don't feel strongly though. If you still feel like it would be more appropriate in `postinstall` I can make the change.

Reply:
Thanks for the explanation. I didn't know that it would install the dependencies and then run the install script. Disregard my comment.